src/hooks/usePerformanceMonitoring.ts

```
@@ -0,0 +1,167 @@
+      if (process.env.NODE_ENV === 'development') {
+        console.group('🎨 Paint Timing');
+        paintTiming.forEach(({ name, startTime }) => {
+          console.log(`${name}:`, `${Math.round(startTime)}ms`);
+        });
+        console.groupEnd();
+      }
+    }
+
+    // Memory usage (Chrome only)
+    if ('memory' in performance) {
+      const memory = (performance as any).memory;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)